README.md

```diff
@@ -1,15 +1,40 @@
 # DataLine
 
-## Running
+## Getting started
 
-To run DataLine, you can use our official docker image:
+To run DataLine, you can use our official docker image and get started in one command:
 ```docker run -p 2222:2222 -p 7377:7377 -v dataline:/home/.dataline --name dataline ramiawar/dataline:v0.1.0```
 
```

Comment:
I'm thinking we could also add the command to stop/start the container. I think it's `docker start/stop dataline`